api-reference/reference.md

```diff
@@ -1563,6 +1563,177 @@ The intended purpose of the file. Currently, only `batch` is supported.
 
 ---
 
+## Embeddings
+
+### Create embeddings
+
+`POST https://api.kluster.ai/v1/embeddings`
+
+Creates an embedding vector representing the input text. These vector representations can be easily consumed by machine learning models and algorithms to understand semantic relationships between pieces of text.
+
+<div class="grid" markdown>
+<div markdown>
+
+**Request**
+
+`input` ++"string or array"++ <span class="required" markdown>++"required"++</span>
+
+Input text to embed, encoded as a string or array of tokens. To embed multiple inputs in a single request, pass an array of strings or array of token arrays. The input must not exceed the max input tokens for the model (8192 tokens for `BAAI/bge-m3`), cannot be an empty string, and any array must be 2048 dimensions or less.
+
+---
+
+`model` ++"string"++ <span class="required" markdown>++"required"++</span>
+
+ID of the model to use.
+
+---
+
+`dimensions` ++"integer"++
+
+The number of dimensions the resulting output embeddings should have.
+
+---
+
+`encoding_format` ++"string"++
+
+The format to return the embeddings in. Can be either `float` or `base64`. Defaults to `float`.
+
+---
+
+`user` ++"string"++
+
+A unique identifier representing your end-user.
+
+---
+
+**Returns**
+
+A list of [embedding objects](#embedding-object).
```

Comment:
```suggestion
A list of [Embedding objects](#embedding-object).
```

Capitalize here because it should match the title of the section we're linking to